base/applications/cmdutils/more/more.c

```diff
@@ -486,6 +790,141 @@ LoadRegistrySettings(HKEY hKeyRoot)
     RegCloseKey(hKey);
 }
 
+static inline BOOL IsFlag(LPCWSTR param)
```

Comment:
This has been moved here, together with the other command-line parsing functions.

Reply:
OK.